assets/css/base.css

```diff
@@ -69,26 +78,57 @@ body {
   color: #2c3e50;
 }
 
-/* Make meetup iframe responsive friendly */
+h2, h3, h4, h5 {
+  margin: 0 0 15px;
+  color: #18bc9c;
```

Comment:
This change affects the title on the footer. The "Vigo Java User Group" and "Síguenos" disappear.